frontend/src/pages/_app.tsx

```diff
@@ -1,8 +1,10 @@
 import { AppProps } from "next/app";
+import Script from "next/script";
 import { QueryCache, ReactQueryCacheProvider } from "react-query";
 import { ReactQueryDevtools } from "react-query-devtools";
 import { checkFeatureFlags } from "src/common/featureFlags";
 import CookieBanner from "src/components/CookieBanner";
+import configs from "src/configs/configs";
 import "src/global.scss";
```

Comment:
is there a certain way we order import statements on our team?

Reply:
Yes! We use `prettier` to sort the imports for us 🙆‍♂️ 

Also in case you don't have it, we actually have `pre-commit` to lint both BE and FE at commit time:

`brew install pre-commit && pre-commit install`!